static/app/views/starfish/views/spans/spansTable.tsx

```diff
@@ -97,12 +97,15 @@ export default function SpansTable({
     });
   };
 
+  const isEndpointOverview = Boolean(endpoint);
```

Comment:
Definitely fine, but maybe hides the intent a bit. Having a `shouldTrackVCD` prop might be better, then it's clearer that this component only tracks its VCD on certain pages

Reply:
good catch, i like that change